lua-5.3.0-idsize.patch

```diff
@@ -0,0 +1 @@
+shared-modules/lua5.3/lua-5.3.0-idsize.patch
```

Comment:
Is this to workaround a bug in ancient `flatpak-builder`? I've never seen anybody else have a problem with patches in submodules.

Reply:
Yes, it's been fixed at least in 0.9.x.